prow/config.yaml

```diff
@@ -224,6 +224,22 @@ presubmits:
         hostPath:
           path: /mnt/disks/ssd0
 
+  kubeflow/examples:
+  - name: kubeflow-examples-presubmit
+    context: kubeflow-examples-presubmit
+    agent: kubernetes
+    always_run: true         # Run for every PR, or only when requested.
+    rerun_command: "/test kubeflow-examples-presubmit"
+    trigger: "(?m)^/test( all| kubeflow-examples-presubmit),?(\\s+|$)"
+    branches:
+    - master
+    labels:
+      preset-service-account: true
+    spec:
+      containers:
+      - image: gcr.io/kubeflow-ci/test-worker:latest
```

Comment:
can you link to the image Dockerfile?

Reply:
The Dockerfile is here
https://github.com/kubeflow/testing/blob/master/images/Dockerfile

What do you mean by link to the Dockerfile? Do you mean add it in a comment?

Reply:
nope, was curious how it works with git checkout, and looking if that's possible to integrate with our checkout procedures

Reply:
How do your git checkout procedures work?